ui/src/app/pages/analysis/notebook-action-menu.spec.tsx

```diff
@@ -1,35 +0,0 @@
-import '@testing-library/jest-dom';
-
-import { WorkspaceAccessLevel } from 'generated/fetch';
-
-import { screen } from '@testing-library/react';
-import { NotebookActionMenu } from 'app/pages/analysis/notebook-action-menu';
-
-import { renderWithRouter } from 'testing/react-test-helpers';
-
-describe('NotebookActionMenu', () => {
-  const props = {
-    notebook: {
-      name: 'name',
-    },
-    permission: WorkspaceAccessLevel.WRITER,
-  };
-  const component = () => {
-    return renderWithRouter(
-      <NotebookActionMenu
-        resource={props}
-        existingNameList={[]}
-        onUpdate={() => {}}
-      />
-    );
-  };
-
-  it('should render', () => {
```

Comment:
this did not seem worthwhile - it would have needed an update due to the signature change